cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -1049,6 +1058,7 @@ def create_xblock_info(  # lint-amnesty, pylint: disable=too-many-statements
     user=None,
     course=None,
     is_concise=False,
+    ai_aside_summary=None,
```

Comment:
this field should probably be something like ai_aside_summary_config or just summary_config, this sounds like it contains the actual summary text

Reply:
in fact. that a current instance of `AiAsideSummary` maybe I can rename the `class` to something more semantic like `AiAsideSummaryConfiguration`

Reply:
Renamed the class to `AiAsideSummaryConfig` and param to `summary_configuration`